src/test/java/ledger/SQLiteTest.java

```diff
@@ -0,0 +1,96 @@
+package ledger;
+
+/**
+ * Created by Tayler How on 9/24/2016.
+ */
+
+import org.junit.*;
+import static org.junit.Assert.*;
+
+import java.sql.*;
+
+public class SQLiteTest {
+    static Connection dbConnection;
+
+    @BeforeClass
+    public static void SetupSQLiteConnection() {
+        dbConnection = null;
+        try {
+            Class.forName("org.sqlite.JDBC");
+            dbConnection = DriverManager.getConnection("jdbc:sqlite:test.db");
+
+            Statement stmt = dbConnection.createStatement();
+            String createTableSQL = "CREATE TABLE COMPANY " +
+                    "(ID INT PRIMARY KEY     NOT NULL," +
+                    " NAME           TEXT    NOT NULL, " +
+                    " AGE            INT     NOT NULL, " +
+                    " ADDRESS        CHAR(50), " +
+                    " SALARY         REAL)";
+            stmt.executeUpdate(createTableSQL);
+            stmt.close();
+        } catch (Exception e) {
+            System.err.println(e.getClass().getName() + ": " + e.getMessage());
+            System.exit(0);
```

Comment:
This will close the test runner and all we will know is that it exited.

Either throw the exception or call Assert.fail().